changelog/14791.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
I feel like other PRs have labeled this as an `improvement`. Do we have a consensus on what we are using to categorize the file-permission changes?